remember.html

```diff
@@ -8,15 +8,15 @@
 	</head>
 	<body>
 		<div class="container">
-			<header>
-				<div class="item1"></div>
-				<div class="item1" style="height: 20px; width: 30px;"></div>
-			</header>
-			<article>
-				<div class="item2 purple"></div>
-				<div class="item2 green"></div>
-				<div class="item2 blue"></div>
-			</article>
-		<div>
+			<div class="header">
+				<div class="header__item"></div>
+				<div class="header__item small"></div>
```

Comment:
"small" is the modifier in this case. So the class must be formed like this: "header__item header__item_small".
The same situation with purple, green, and blue.